edgraph/server.go

```diff
@@ -1573,6 +1613,62 @@ func parseRequest(qc *queryContext) error {
 	return validateQuery(qc.dqlRes.Query)
 }
 
+// addQueryIfUniqueTrue modifies the request if @unique defined on Predicate
```

Comment:
addQueryIfUniqueTrue -> addQueryIfUnique

addQueryIfUniqueTrue adds dummy queries in the request for checking whether predicate is unique in the db